spec/readme.md

```diff
@@ -10,7 +10,7 @@ This tar file can be created from most common container tools (e.g., Docker).
 
 ## The base folder
 
-To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.
+To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.<br>
```

Comment:
Agreed that it's weird. I only ever use it for manual line breaks - purely a cosmetic change.
If only there were better ways:

* A blank line separating them would be interpreted as a new paragraph
* The [CommonMark spec on hard line breaks](http://spec.commonmark.org/0.27/#hard-line-breaks) offers two trailing spaces, but a lot of tools consider trailing whitespace to be an error - indeed, I have my editor set to automatically trim!
* The other option it offers is a trailing backslack - `foo\` - but github does not seem to have adopted CommonMark yet and that markup does not render correctly

So basically, I use `<br>` sparingly when the Github readme render would look nicer, with the assumption that we can switch to the non-HTML backslash when CommonMark takes over the world. That cool with you? :sunglasses: